external/vsphere-policy/api/v1alpha1/computepolicy_types.go

```diff
@@ -0,0 +1,99 @@
+// © Broadcom. All Rights Reserved.
```

Comment:
Are you not using the standard copyright (with year) ? -> https://github-vcf.devops.broadcom.net/vcf/vmware-git_tools/blob/eac5adec3918d2b3e369ce29b11f72f1e49590ac/src/git_tools/update_copyrights.py#L71
```
// Copyright (c) 2025 Broadcom. All Rights Reserved.
// Broadcom Confidential. The term "Broadcom" refers to Broadcom Inc.
// and/or its subsidiaries.
// SPDX-License-Identifier: Apache-2.0
```

Reply:
Because not all projects do (see GoVmomi) and this way we do not have to routinely update the year.